gadget/install/install.go

```diff
@@ -232,6 +232,15 @@ func createPartitions(model gadget.Model, gadgetRoot, kernelRoot, bootDevice str
 
 	// auto-detect device if no device is forced
 	// device forcing is used for (spread) testing only
+	if bootDevice == "" {
+		foundDisk, err := disks.DiskFromMountPoint("/run/mnt/ubuntu-seed", nil)
+		if err == nil {
+			bootDevice = foundDisk.KernelDeviceNode()
+		} else {
+			logger.Noticef("Warning: cannot find disk from mounted seed: %s", err)
+			bootDevice = ""
```

Comment:
Unless I miss something `bootDevice`is already `""` (check above in line 235)(?)